git-tar/function/ops.go

```diff
@@ -372,6 +373,7 @@ func deploy(tars []tarEntry, pushEvent sdk.PushEvent, stack *stack.Services, sta
 		httpReq.Header.Add("Sha", afterCommitID)
 		httpReq.Header.Add("Scm", sourceManagement)
 		httpReq.Header.Add("Private", strconv.FormatBool(privateRepo))
+		httpReq.Header.Add("Repo_URL", repositoryURL)
```

Comment:
Think you meant to put: `Repo-URL` here. The underscores are converted by the watchdog.